.config/autoconfig.php

```diff
@@ -1,38 +1,26 @@
 <?php
 
+require_once "util.php";
+
 $autoconfig_enabled = false;
 
-if (getenv('SQLITE_DATABASE')) {
+if (getConfigFromEnv('SQLITE_DATABASE')) {
```

Comment:
database config from env also needs supported in the entrypoint script too. sqlite db isn't in there yet.